File: patterns/c-patterns/pattern134.c
// Pattern 134: ...

// *       *
// *       *
// * * * * *
//         *
//         *

#include <stdio.h>

int main()
{
    int n = 5;
    int mid = n / 2 + 1;
    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n; j++)
        {
            if (i == mid || (j == 1 && i <= mid) || j == n)
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern134.js
// Pattern 134: ...

// *       *
// *       *
// * * * * *
//         *
//         *

let n = 5;
let mid = Math.floor(n / 2) + 1;

for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j <= n; j++) {
        if (i === mid || (j === 1 && i <= mid) || j === n) {
            row += '* ';
        } else {
            row += '  ';
        }
    }
    console.log(row);
}
